http.go

```diff
@@ -43,6 +44,11 @@ func processRequest(tx types.Transaction, r *http.Request) (*types.Interruption,
 		tx.AddRequestHeader(k, v[0])
 	}
 	tx.AddRequestHeader("Host", r.Host)
+	serverName, err := parseServerName(r.Host)
+	if err != nil {
+		// TODO: log a warning, parseServerName still populates serverName
```

Comment:
If internally an error is raised, r`.Host` is still returned to populate the ServerName. So a debug level log may be enough. How could I raise it from `processRequest`?

Reply:
Pass a logger to processRequest.

Reply:
Done, thanks!